lithops/standalone/proxy.py

```diff
@@ -60,12 +93,11 @@ def init_keeper():
     config_file = os.path.join(REMOTE_INSTALL_DIR, 'config')
     with open(config_file, 'r') as cf:
         serverfull_config = json.load(cf)
-    handler = StandaloneHandler(serverfull_config)
 
-    if handler.auto_dismantle:
-        keeper = threading.Thread(target=budget_keeper, args=(handler,))
-        keeper.daemon = True
-        keeper.start()
+    handler = StandaloneHandler(serverfull_config)
```

Comment:
by removing this the VM will be always stopped, either by hard or soft timeout. don't we want to have a way to control it by forcing auto_dismantle to False, while having it True by default?

Reply:
The idea is avoid VMs started forever. Users could have auto dismantle set to false and then never explicitly call the exec.dismantle() method. IMO we should ensure VMs are always stopped and hard_timeout is a good way to do this